packages/api/models/schemas.js

```diff
@@ -145,7 +145,7 @@ module.exports.collection = {
           },
           fileType: {
             title: 'File Type',
-            description: 'CNM Filetype.  Used for submission to CMR.  Non CNM Filetypes default to "data"',
+            description: 'CNM Filetype.  Cumulus uses this for CMR submission.  Non-CNM filetypes will be treated as "data" fileType',
```

Comment:
```suggestion
            description: 'CNM file type.  Cumulus uses this for CMR submission.  Non-CNM file types will be treated as "data" fileType',
```

I may be totally wrong, but do we only want to use the compound variable name  `fileType` when we're explicitly referring to the variable?

Reply:
👍  excellent point.